apps/ledger-live-desktop/src/renderer/Default.jsx

```diff
@@ -250,6 +251,7 @@ export default function Default() {
                             />
                             <Route path="/lend" render={props => <Lend {...props} />} />
                             <Route path="/exchange" render={(props: any) => <Exchange />} />
+                            <Route path="/earn" render={props => <Earn {...props} />} />
```

Comment:
Should we wrap this in the same FeatureToggle (ptxEarn) as well in order to prevent anyone from accessing the screen via a deep link?

Reply:
Done